TROUBLESHOOTING.md

```diff
@@ -11,11 +11,14 @@ that gets printed to stderr.
 
 <!-- BEGIN_GOOGLE_INTERNAL_DOCUMENTATION
 GRPC_VERBOSITY has been disabled for internal usage and will not work anymore.
-If anyone wants to debug, we need to set verbose logs using absl.
+If anyone wants to debug, we need to [set log verbosity using absl](https://abseil.io/docs/cpp/guides/logging).
+
 END_GOOGLE_INTERNAL_DOCUMENTATION -->
 
 <!-- BEGIN_OPEN_SOURCE_DOCUMENTATION -->
-`GRPC_VERBOSITY` is used to set the minimum level of log messages printed by gRPC (supported values are `DEBUG`, `INFO` and `ERROR`). If this environment variable is unset, only `ERROR` logs will be printed. `ERROR` is recomeded for production systems.
+Our recommendation is to avoid using this flag and [set log verbosity using absl](https://abseil.io/docs/cpp/guides/logging). We only support this flag for legacy reasons.
```

Comment:
`GRPC_VERBOSITY` should not be a top level section anymore. It doesn't make sense to start off by saying "Don't do this". It should instead say - "Do this".

And then, as a side-note, you can talk about the legacy support for `GRPC_VERBOSITY`.

Reply:
Changed everything.